iFixit/react-commerce
# Add header navigation

The PR introduces a react based header bar (not fully featured yet, some parts like the cart and the workbench have been postponed).

The menu data is fetched from Strapi, so menu links can be added, removed or changed. Right now the UI supports a main menu with 3 submenus. In the future it could be extended to support other combination if needed (e.g. link with image).

The header doesn't reflect exactly the current iFixit header, since it introduces a few changes (e.g. "login" / "join" links have been removed in favor of a default user avatar that let us avoid CLS, since auth data is fetched client side).

## QA

1. Visit the Vercel PR preview
2. Visit parts product list page and copy the url
3. Verify that #128 requirements under "Do this" are met

closes #128
